src/views/Change.vue

```diff
@@ -54,6 +54,8 @@
           class="mt-10"
           sectionNumber="3."
           :validate="getAppValidate"
+          :invalidSection="isCertifyInvalid"
+          :disableEdit="!isRoleStaff"
```

Comment:
These 2 props don't match the prop declarations in the CertifySection component.

And while you're here, I had the idea that this page (Change.vue) doesn't need to know if certify is invalid, so you could move that logic into the CertifySection component.

Reply:
Does that mean that disableEdit should not be set with isRoleStaff?

Reply:
I changed the prop declaration in CertifySection component.  @Prop({ required: false }) readonly disableEdit: boolean =>  @Prop({ required: true }) readonly disableEdit!: boolean. I think this fixes disableEdit. For isCertifyIsInvalid I removed it and let the get invalidSection in the CertifySection component deal with the logic.

Reply:
My comment was that you were passing in 2 props that didn't have `@Prop` declarations in the sub-component. I see you've changed the code since that comment.

No, I think it's correct that edit should be disabled except for staff.

I'm resolving this conversation so I can comment cleanly on your new code.